osu.Game/Beatmaps/BeatmapMetadata.cs

```diff
@@ -29,6 +29,9 @@ public class BeatmapMetadata : IEquatable<BeatmapMetadata>, IHasPrimaryKey
         [JsonProperty("artist_unicode")]
         public string ArtistUnicode { get; set; }
 
+        private string artistUnicode => string.IsNullOrEmpty(ArtistUnicode) ? Artist : ArtistUnicode;
+        private string titleUnicode => string.IsNullOrEmpty(TitleUnicode) ? Title : TitleUnicode;
```

Comment:
I don't think these have to be properties? They're named in a potentially misleading way, too. Just variables in `ToRomanisableString()` would do fine, no?

Reply:
Yeah maybe, I'll move it